tests/CMakeLists.txt

```diff
@@ -40,3 +40,4 @@ add_subdirectory(userclk)
 add_subdirectory(fpgametrics)
 add_subdirectory(libofs)
 add_subdirectory(ofs_driver)
+add_subdirectory(ofs_cpeng)
```

Comment:
Does this test compile/pass if `OPAE_BUILD_OFS_CPENG` is not true?

Reply:
Probably not, I'll fix that.